site/sfguides/src/chalice-native-app-quickstart/chalice-native-app-quickstart.md

```diff
@@ -0,0 +1,139 @@
+summary: "A concise guide to demonstrate the use of the Chalice TVB Native App to detect overbidding in programmatic auctions."
+id: "chalice-native-app-quickstart"
+categories: data-science, native-app
+environments: "web"
+status: "Published"
+feedback_link: "https://github.com/Snowflake-Labs/sfguides/issues"
+tags: Getting Started, Data Science, Chalice, Native App 
+authors: Tylynn Pettrey, Tucker Ward 
+
+# Chalice Native App Guide
```

Comment:
@Typettrey Change the title to "Getting Started With Chalice Native App in Snowflake"